stat_fs.go

```diff
@@ -0,0 +1,66 @@
+package hdfs
+
+import (
+	"errors"
+
+	hdfs "github.com/colinmarc/hdfs/protocol/hadoop_hdfs"
+)
+
+var StatFsError = errors.New("Failed to get HDFS usage")
+
+// FsInfo provides information about HDFS
+type FsInfo struct {
```

Comment:
There's no interface to implement, so I'd rather just make these exported and drop the getter methods.